control/installation.SLES.xml

```diff
@@ -35,66 +35,117 @@ defined by a role can be overridden in the next steps if necessary.&lt;/p&gt;</l
-        <root_max_size>10GB</root_max_size>
-        <proposal_lvm config:type="boolean">false</proposal_lvm>
-        <vm_desired_size>15GB</vm_desired_size>
-        <vm_home_max_size>25GB</vm_home_max_size>
-        <btrfs_increase_percentage config:type="integer">300</btrfs_increase_percentage>
-        <!-- the default subvolume "@" was requested by product management -->
-        <btrfs_default_subvolume>@</btrfs_default_subvolume>
-
-        <!-- Subvolumes to be created for a Btrfs root file system -->
-        <!-- copy_on_write is true by default -->
-        <!-- The XML parser is very simplistic, thus not using attributes, but sub-elements -->
-        <subvolumes config:type="list">
-            <subvolume>
-                <path>home</path>
-            </subvolume>
-            <subvolume>
-                <path>opt</path>
-            </subvolume>
-            <subvolume>
-                <path>srv</path>
-            </subvolume>
-            <subvolume>
-                <path>tmp</path>
-            </subvolume>
-            <subvolume>
-                <path>usr/local</path>
-            </subvolume>
-            <!-- unified var subvolume - https://lists.opensuse.org/opensuse-packaging/2017-11/msg00017.html -->
-            <subvolume>
-                <path>var</path>
-                <copy_on_write config:type="boolean">false</copy_on_write>
-            </subvolume>
-
-            <!-- architecture specific subvolumes -->
-
-            <subvolume>
-                <path>boot/grub2/i386-pc</path>
-                <archs>i386,x86_64</archs>
-            </subvolume>
-            <subvolume>
-                <path>boot/grub2/x86_64-efi</path>
-                <archs>x86_64</archs>
-            </subvolume>
-            <subvolume>
-                <path>boot/grub2/powerpc-ieee1275</path>
-                <archs>ppc,!board_powernv</archs>
-            </subvolume>
-            <subvolume>
-                <path>boot/grub2/x86_64-efi</path>
-                <archs>x86_64</archs>
-            </subvolume>
-            <subvolume>
-                <path>boot/grub2/s390x-emu</path>
-                <archs>s390</archs>
-            </subvolume>
-        </subvolumes>
+       <expert_partitioner_warning config:type="boolean">false</expert_partitioner_warning>
+
+        <proposal>
+            <lvm config:type="boolean">false</lvm>
+            <proposal_settings_editable config:type="boolean">true</proposal_settings_editable>
+            <root_subvolume_read_only config:type="boolean">false</root_subvolume_read_only>
+        </proposal>
+
+        <volumes config:type="list">
+            <!-- / volume: 3 GiB - 10 GiB, *4 if snapshots are used -->
+            <volume>
+                <mount_point>/</mount_point>
+                <fs_type>btrfs</fs_type>
+
+                <desired_size config:type="disksize">5 GiB</desired_size>
+                <min_size config:type="disksize">3 GiB</min_size>
```

Comment:
this slightly depends on https://bugzilla.suse.com/show_bug.cgi?id=1072247#c6

Reply:
Well, better get SLES in with current settings, adjusting the parameters can be done any time later.

Reply:
Not for me to change; I've kept the existing min values here.